src/box/tuple.h

```diff
@@ -261,6 +259,7 @@ box_tuple_extract_key(const box_tuple_t *tuple, uint32_t space_id,
  *
  * Each 'off_i' is the offset to the i-th indexed field.
  */
+
```

Comment:
No, please, remove it. Comment about a structure must be placed right before the structure. Most of autotools for documentation generating assume this.